velox/type/Timestamp.cpp

```diff
@@ -166,9 +165,24 @@ void appendSmallInt(int n, std::string& out) {
   out.append(intToStr[n], 2);
 }
 
+std::string::size_type getCapacity(const TimestampToStringOptions& options) {
+  int width = static_cast<int>(options.precision);
+  switch (options.mode) {
+    case TimestampToStringOptions::Mode::kDateOnly:
+      return 10;
```

Comment:
Presume this comes from yyyy-mm-dd ? Please can you explicitly say so.

Reply:
yes, I'll add this to the comment.